src/formpack/utils/expand_content.py

```diff
@@ -180,25 +208,27 @@ def _mark_special(**kwargs):
             _mark_special(column_name=column_name,
                           column=column_name,
                           translation=UNTRANSLATED)
-        if ':' not in column_name:
+        if ':' not in column_name and column_name not in MEDIA_TYPES:
             continue
         if column_name.startswith('bind:'):
             continue
         if column_name.startswith('body:'):
             continue
-        mtch = re.match(r'^media\s*::?\s*([^:]+)\s*::?\s*([^:]+)$', column_name)
+        mtch = re.match(rf'^(media\s*::?\s*)?({RE_MEDIA_TYPES})\s*::?\s*([^:]+)$', column_name)
         if mtch:
             matched = mtch.groups()
-            media_type = matched[0]
+            media_type = matched[-2]
+            translation = matched[-1]
```

Comment:
Why using negative indexes? Starting from the beginning should work? 

- `media::image::foo` 
  1. `group(0)` -> media::image::foo
  1. `group(1)` -> media
  1. `group(2)` -> image  
  1. `group(3)` -> foo

- `image::foo` 
  1. `group(0)` -> image::foo
  1. `group(1)` -> None
  1. `group(2)` -> image
  1. `group(3)` -> foo

Reply:
Aha! That's much better ❤️ For some reason I assumed this wasn't the case but never tested it 🙃